pyquil/device/transformers/graph_to_compiler_isa.py

```diff
@@ -0,0 +1,158 @@
+from typing import Union
+
+import numpy as np
+from typing import List, Optional, cast
+from pyquil.external.rpcq import (
+    GateInfo,
+    MeasureInfo,
+    Supported1QGate,
+    Supported2QGate,
+    CompilerISA,
+    add_qubit,
+    add_edge,
+)
+
+import networkx as nx
+
+
+DEFAULT_1Q_GATES = [
+    Supported1QGate.I,
+    Supported1QGate.RX,
+    Supported1QGate.RZ,
+    Supported1QGate.MEASURE,
+]
+DEFAULT_2Q_GATES = [
+    Supported2QGate.CZ,
+    Supported2QGate.XY,
+]
+
+
+def graph_to_compiler_isa(
+    graph: nx.Graph, gates_1q: Optional[List[str]] = None, gates_2q: Optional[List[str]] = None
+) -> CompilerISA:
+    """
+    Generate an ``CompilerISA`` object from a NetworkX graph and list of 1Q and 2Q gates.
+    May raise ``GraphGateError`` if the specified gates are not supported.
+
+    :param graph: The graph topology of the device.
+    :param gates_1q: A list of 1Q gate names to be applied to all qubits in the device.
```

Comment:
`to be applied` -> `to be made available for`. I can understand the intent of what it's saying but I think it's better to be clear here.